apex_dual_test.go

```diff
@@ -14,19 +13,19 @@ import (
 
 func setupEmptyLookupFuncs(gw *Gateway) {
 	if resource := gw.lookupResource("HTTPRoute"); resource != nil {
-		resource.lookup = func(_ []string) []netip.Addr { return []netip.Addr{} }
+		resource.lookup = func(_ []string) map[string][]string { return map[string][]string{} }
```

Comment:
Just why?

Reply:
To include TXT records that are not ip addresses